src/transformers/integrations/integration_utils.py

```diff
@@ -960,9 +960,9 @@ def setup(self, args, state, model):
             remote server, e.g. s3 or GCS. If set to `True` or *1*, will copy each saved checkpoint on each save in
             [`TrainingArguments`]'s `output_dir` to the local or remote artifact storage. Using it without a remote
             storage will just copy the files to your artifact location.
-        - **MLFLOW_TRACKING_URI** (`str`, *optional*, defaults to `""`):
-            Whether to store runs at a specific path or remote server. Default to an empty string which will store runs
-            at `./mlruns` locally.
+        - **MLFLOW_TRACKING_URI** (`str`, *optional*, defaults to `None`):
```

Comment:
The convention in the library is not to specify `None` as a default

```suggestion
        - **MLFLOW_TRACKING_URI** (`str`, *optional*):
```